src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -242,6 +241,13 @@ export default {
 		this.updateFromParams(to.query);
 		next();
 	},
+	beforeRouteLeave(to, from, next) {
+		if (typeof window !== 'undefined' && to.path.indexOf('/lend/') !== -1) {
```

Comment:
is window required? are we not expecting this to be called on the server?

Reply:
I only want this called when we click a router link in the client. no server redirect for this.

Reply:
👍